stl/inc/xutility

```diff
@@ -5307,6 +5442,11 @@ namespace ranges {
 
     inline constexpr _Find_if_not_fn find_if_not{_Not_quite_object::_Construct_tag{}};
 } // namespace ranges
+
+// STRUCT TEMPLATE _Require_constant
+template <auto>
+struct _Require_constant; // not defined; _Require_constant<E> is a valid type if E is a constant expression of
+                          // structural type

```

Comment:
This could more precisely mirror the language of http://eel.is/c++draft/temp.names#7:
```suggestion
struct _Require_constant; // _Require_constant<E> is a valid template-id iff E is a constant expression of structural type
```

Reply:
That would look good to me.